test/e2e/atlas/backup_compliancepolicy_setup_test.go

```diff
@@ -46,9 +43,11 @@ func TestBackupCompliancePolicySetup(t *testing.T) {
 		RetentionValue:    1,
 	}
 	policy := &atlasv2.DataProtectionSettings20231001{
-		ScheduledPolicyItems: []atlasv2.BackupComplianceScheduledPolicyItem{scheduledPolicyItem},
-		ProjectId:            &g.projectID,
-		AuthorizedEmail:      authorizedEmail,
+		ScheduledPolicyItems:    []atlasv2.BackupComplianceScheduledPolicyItem{scheduledPolicyItem},
+		ProjectId:               &g.projectID,
+		AuthorizedUserLastName:  authorizedUserFirstName,
```

Comment:
this is making the test red @blva 
```suggestion
		AuthorizedUserLastName:  authorizedUserLastName,
		AuthorizedUserFirstName: authorizedUserFirstName,
```